src/extensions/ffxiv_weather.py

```diff
@@ -0,0 +1,67 @@
+from discord.ext import commands
+import discord
+
+import datetime
+import ffxivweather
+
+list = ["Limsa Lominsa", "Middle La Noscea", "Lower La Noscea", 
```

Comment:
This list is only limited to zones within Limsa Lominsa (which is 1/8th of the available zones). We should include every existing region.